packages/evolution-frontend/tests/ui-testing/testHelpers.ts

```diff
@@ -347,6 +374,31 @@ export const inputRangeTest: InputRangeTest = ({ context, path, value, sliderCol
     });
 };
 
+/**
+ * Test to check if the checkboxes for a given path are working as expected.
+ * It iterates over the provided values, checks each checkbox, and verifies that it is checked.
+ *
+ * @param {object} params - The parameters for the test.
+ * @param {string} params.path - The path to the checkboxes to be tested.
+ * @param {(string|number)[]} params.values - The values to be checked in the checkboxes.
+ */
+export const inputCheckboxTest: InputCheckboxTest = ({ context, path, values }) => {
+    test(`Check ${values} for ${path} - ${getTestCounter(context, `${path} - ${values}`)}`, async () => {
+        const newPath = context.objectDetector.replaceWithIds(path);
+        const resolvedValues = values.map((value) =>
+            typeof value === 'string' ? context.objectDetector.replaceWithIds(value) : value
+        );
+
+        for (const value of resolvedValues) {
+            const checkbox = context.page.locator(`id=survey-question__${newPath}_${value}__input-checkbox__${value}`);
+            await checkbox.scrollIntoViewIfNeeded();
+            await checkbox.click();
+            await expect(checkbox).toBeChecked();
```

Comment:
Je me demande s'il ne faudrait pas checker l'état checked à la toute fin seulement? (tout d'in coup qu'on s'attendait à avoir 3 valeurs checked, mais que le widget a "accidentellement" décoché les valeurs cochées précédemment...)

En fait, on peut garder le check ici, et après la boucle for, il faudrait aller chercher les éléments checked et valider qu'ils correspondent à la valeur. Tu peux juste mettre un TODO ici au pire.

Reply:
Bonne idée. C'est fait.